meson.build

```diff
@@ -365,30 +365,27 @@ endif
 
 if build_standalone and get_option('plugin_uefi_capsule')
   efiboot = dependency('efiboot')
-  objcopy = find_program ('objcopy')
-  genpeimg = find_program ('genpeimg', required: false)
-
-  efi_app_location = join_paths(libexecdir, 'fwupd', 'efi')
-  conf.set_quoted ('EFI_APP_LOCATION', efi_app_location)
 
   if host_cpu == 'x86'
           EFI_MACHINE_TYPE_NAME = 'ia32'
-          gnu_efi_arch = 'ia32'
   elif host_cpu == 'x86_64'
           EFI_MACHINE_TYPE_NAME = 'x64'
-          gnu_efi_arch = 'x86_64'
   elif host_cpu == 'arm'
           EFI_MACHINE_TYPE_NAME = 'arm'
-          gnu_efi_arch = 'arm'
   elif host_cpu == 'aarch64'
           EFI_MACHINE_TYPE_NAME = 'aa64'
-          gnu_efi_arch = 'aarch64'
   else
           EFI_MACHINE_TYPE_NAME = ''
-          gnu_efi_arch = ''
   endif
   conf.set_quoted('EFI_MACHINE_TYPE_NAME', EFI_MACHINE_TYPE_NAME)
 
+  efi_name = 'fwupd@0@.efi'.format(EFI_MACHINE_TYPE_NAME)
+  efi_app_location = join_paths(libexecdir, 'fwupd', 'efi', efi_name)
+  fs = import('fs')
+  if not fs.exists (efi_app_location)
```

Comment:
does `fs` exist in the lowest meson version we support? I thought that was kinda new...

Reply:
What's the minimum version we realistically need to target with 1.6.0?  You're right it's new, but maybe with 1.6.0 we can bump higher?

Reply:
True; 1.6.0 seems like a nice line in the sand.